moc_openshift.py

```diff
@@ -117,7 +134,7 @@ def user_rolebinding_exists(self, user_name, project_name, role):
 
         result = self.get_rolebindings(project_name, openshift_role)
         if result.status_code == 200 or result.status_code == 201:
-            role_binding = result.json()
+            role_binding = result.json
```

Comment:
This is a `requests.Reponse` so the `.json` is a method, not a property. Add back the parenthesis.